Annotations.md

```diff
@@ -0,0 +1,134 @@
+# Acquia RA composer.json Template w/Annotations
+[Name and Description](#name-and-description)
+[Type](#type)  
```

Comment:
Weird. Is "Type" missing in the rendered TOC for anyone else?: https://github.com/acquia/acquia-ra-composer/blob/7905cca47c66e5fa453aec7a6ff0b2b219e97e96/Annotations.md

Reply:
Preposterous!

(Needed a double space after the Name and Description line)